[2021.11.30/10:33:50] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 383, in ppaButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 269, in changeImages
    tiem.sleep(1)
NameError: name 'tiem' is not defined

[2021.11.30/11:06:46] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 349, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 260, in changeImages
    self.OffOrDef = self.showImageGroupBox("", self.defenceImage, 90, 90)
AttributeError: 'Game' object has no attribute 'showImageGroupBox'

[2021.11.30/11:06:47] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in jjiButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 283, in changeImages
    self.playerLastShape = self.showImageGroupBox("player", self.jjiImage, 160, 160)
AttributeError: 'Game' object has no attribute 'showImageGroupBox'

[2021.11.30/11:41:22] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:26] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:27] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:27] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:28] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:29] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:29] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:30] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:36] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:38] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:41] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:41] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:41] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:41] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:42] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:42] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:42] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:42] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:42] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:41:43] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:29] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:30] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:30] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:30] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:30] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:31] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:31] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:31] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:31] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:31] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:32] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:32] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:32] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:33] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:33] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:34] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:34] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:35] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:35] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:35] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:35] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 358, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:36] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:36] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:37] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 327, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:42:38] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 343, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:45:55] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 333, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:45:57] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 333, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 333, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 333, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 333, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 333, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 333, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 348, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 348, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 348, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 348, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 362, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 362, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 362, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 348, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 348, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 333, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:46:05] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 333, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:05] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:05] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:47:05] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:48:58] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:48:58] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:48:59] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:48:59] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:48:59] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:48:59] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:48:59] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:00] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:00] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:00] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:00] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:00] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:19] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:19] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:19] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:20] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:20] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:20] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:21] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:49:21] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:17] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:18] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:18] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:18] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:22] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:22] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:22] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:38] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:38] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:41] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:41] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:41] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:41] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:41] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:42] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:42] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:42] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:42] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:43] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:43] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:43] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:43] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:43] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:44] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:44] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:44] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:44] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:44] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:50:44] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:56:19] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:56:19] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:56:20] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:56:20] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 363, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:56:20] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 363, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/11:56:20] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 363, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/12:09:57] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/12:09:58] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/12:10:33] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/12:10:33] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 335, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/12:10:34] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/12:10:34] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 363, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/15:53:55] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 349, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/15:53:55] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/15:53:56] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 377, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:15:51] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:15:52] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:15:53] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:15:54] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 378, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:15:55] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:15:55] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:15:56] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:15:56] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:15:57] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:15:59] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:16:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:16:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 378, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:16:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:16:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:16:38] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:16:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:31] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:33] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:35] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:36] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:36] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:36] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:36] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:37] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:37] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:38] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:38] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:38] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:38] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:24:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:25:29] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:25:30] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:25:30] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:25:31] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:25:32] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:25:32] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:25:32] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:25:32] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:25:33] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:25:33] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:25:33] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:26:59] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 368, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:03] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 381, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:47] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:47] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:47] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 353, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:47] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 367, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:48] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 367, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:48] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 380, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:48] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 380, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:27:50] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 367, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:32:06] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 352, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:38:54] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:38:59] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:39:00] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 377, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:39:00] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 377, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:39:00] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 377, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:39:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 377, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:39:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:39:01] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:39:02] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:40:17] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:40:17] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:40:17] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:40:17] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:40:17] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:40:18] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:40:18] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 364, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:58:48] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 338, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:58:49] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 352, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:58:50] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:58:51] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 338, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:58:51] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 352, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:58:52] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:58:53] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 338, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:58:53] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:58:53] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 365, in ppaButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/16:58:53] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 352, in jjiButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/17:00:58] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 339, in mukButtonClicked
    gameLoop = game.mukjjippa()
AttributeError: 'Game' object has no attribute 'mukjjippa'

[2021.11.30/17:03:37] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 340, in mukButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
TypeError: ingame() missing 1 required positional argument: 'com_dataList'

[2021.11.30/17:04:36] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 355, in jjiButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
TypeError: ingame() missing 1 required positional argument: 'com_dataList'

[2021.11.30/17:04:38] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 369, in ppaButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
TypeError: ingame() missing 1 required positional argument: 'com_dataList'

[2021.11.30/23:34:18] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 359, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 272, in changeImages
    self.lastShape[0] = self.mukImage
IndexError: list assignment index out of range

[2021.11.30/23:34:21] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 359, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 272, in changeImages
    self.lastShape[0] = self.mukImage
IndexError: list assignment index out of range

[2021.11.30/23:34:22] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 374, in jjiButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 286, in changeImages
    self.lastShape[0] = self.jjiImage
IndexError: list assignment index out of range

[2021.11.30/23:34:23] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 388, in ppaButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 300, in changeImages
    self.lastShape[0] = self.ppaImage
IndexError: list assignment index out of range

[2021.11.30/23:34:26] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 374, in jjiButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 286, in changeImages
    self.lastShape[0] = self.jjiImage
IndexError: list assignment index out of range

[2021.11.30/23:53:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 350, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 273, in changeImages
    self.lastShape[0] = self.mukImage
AttributeError: 'Game' object has no attribute 'lastShape'

[2021.11.30/23:55:41] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 352, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 284, in changeImages
    self.lastShape[1] = self.mukImage
AttributeError: 'Game' object has no attribute 'lastShape'

[2021.11.30/23:56:04] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 352, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 284, in changeImages
    self.lastShape[1] = self.mukImage
AttributeError: 'Game' object has no attribute 'lastShape'

[2021.11.30/23:56:18] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 352, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 284, in changeImages
    self.lastShape[1] = self.mukImage
AttributeError: 'Game' object has no attribute 'lastShape'

[2021.11.30/23:56:28] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 352, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 284, in changeImages
    self.lastShape[1] = self.mukImage
AttributeError: 'Game' object has no attribute 'lastShape'

[2021.11.30/23:58:44] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 352, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 284, in changeImages
    self.lastShape[1] = self.mukImage
AttributeError: 'Game' object has no attribute 'lastShape'

[2021.11.30/23:59:51] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 352, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 284, in changeImages
    self.lastShape[1] = self.mukImage
AttributeError: 'Game' object has no attribute 'lastShape'

[2021.12.01/00:08:31] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 346, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 279, in changeImages
    self.lastShape[1] = self.mukImage
AttributeError: 'Game' object has no attribute 'lastShape'

[2021.12.01/00:39:37] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI2.py", line 329, in mukButtonClicked
    self.changeImages(result)
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI2.py", line 257, in changeImages
    self.playerShape.setPixmap(QPixmap(self.mukImage))
AttributeError: 'QGroupBox' object has no attribute 'setPixmap'

[2021.12.01/02:25:50] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 390, in jjiButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: unsupported operand type(s) for +: 'QInformationGroupBox' and 'int'

[2021.12.01/02:25:51] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 404, in ppaButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: unsupported operand type(s) for +: 'QInformationGroupBox' and 'int'

[2021.12.01/02:25:52] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 375, in mukButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: unsupported operand type(s) for +: 'QInformationGroupBox' and 'int'

[2021.12.01/02:25:53] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 390, in jjiButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: unsupported operand type(s) for +: 'QInformationGroupBox' and 'int'

[2021.12.01/02:25:53] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 404, in ppaButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: unsupported operand type(s) for +: 'QInformationGroupBox' and 'int'

[2021.12.01/02:25:54] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 390, in jjiButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: unsupported operand type(s) for +: 'QInformationGroupBox' and 'int'

[2021.12.01/02:25:57] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 375, in mukButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: unsupported operand type(s) for +: 'QInformationGroupBox' and 'int'

[2021.12.01/02:26:00] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 404, in ppaButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: unsupported operand type(s) for +: 'QInformationGroupBox' and 'int'

[2021.12.01/02:26:33] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 390, in jjiButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: unsupported operand type(s) for +: 'QInformationGroupBox' and 'int'

[2021.12.01/02:26:37] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 404, in ppaButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: unsupported operand type(s) for +: 'QInformationGroupBox' and 'int'

[2021.12.01/02:26:39] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 375, in mukButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: unsupported operand type(s) for +: 'QInformationGroupBox' and 'int'

[2021.12.01/02:32:24] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 395, in jjiButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: can only concatenate str (not "int") to str

[2021.12.01/02:32:33] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 409, in ppaButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: can only concatenate str (not "int") to str

[2021.12.01/02:33:25] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 395, in jjiButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: can only concatenate str (not "int") to str

[2021.12.01/02:33:31] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 409, in ppaButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: can only concatenate str (not "int") to str

[2021.12.01/02:33:45] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 395, in jjiButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: can only concatenate str (not "int") to str

[2021.12.01/02:33:47] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 380, in mukButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: can only concatenate str (not "int") to str

[2021.12.01/02:38:18] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 395, in jjiButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: can only concatenate str (not "int") to str

[2021.12.01/02:38:20] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 380, in mukButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: can only concatenate str (not "int") to str

[2021.12.01/02:40:54] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 410, in ppaButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: can only concatenate str (not "int") to str

[2021.12.01/02:41:18] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 395, in jjiButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
  File "C:\Users\lsm12\ad_SWP2\GameUI\game.py", line 114, in ingame
    current_score = score + 2**now
TypeError: can only concatenate str (not "int") to str

[2021.12.01/23:47:31] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 386, in jjiButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
TypeError: ingame() takes 4 positional arguments but 6 were given

[2021.12.01/23:47:40] - Traceback (most recent call last):
  File "C:\Users\lsm12\ad_SWP2\GameUI\gameUI.py", line 400, in ppaButtonClicked
    result = gameLoop.ingame(self.ad_status, shape, self.score, self.now, self.com_dataList)
TypeError: ingame() takes 4 positional arguments but 6 were given

